shared_tests/test_cases.yml

```diff
@@ -16,20 +15,23 @@ book-server_deps:
   - linux
   - quarkus
   withDeps: true
-#book_server_source:
-#  description: book-store basic source analysis
-#  application:
-#    name: book-server
-#    repository:
-#      url: https://github.com/ibraginsky/book-server
-#      kind: git
-#  input: https://github.com/ibraginsky/book-server  # TODO: discuss and unify input application definition
-#  filename: book-server
-#  sources:
-#  - java-ee
-#  - springboot
-#  targets:
-#  - cloud-readiness
-#  - linux
-#  - quarkus
-#  withDeps: false
+  referencesDir: book-server_deps
```

Comment:
I think this is redundant, `book-server_deps` name of the test should be really enough.

Reply:
I don't understand why there is still the `referencesDir`. I think code complexity in parsing is not problem (tested code locally and it worked fine) and it is duplicate with the test_case key field.

Reply:
ah, forgot to remove it, one sec